composer.json

```diff
@@ -51,4 +52,4 @@
 			]
 		}
 	}
-}
+}
```

Comment:
I have my editor set to add a newline at the end of files, not a blocker but I think it's something we default to more broadly in the coding standards, it avoids superfluous changes like this one.

Reply:
```suggestion
}

```